plugin/src/main/java/io/fabric8/maven/plugin/mojo/build/ResourceMojo.java

```diff
@@ -497,14 +479,14 @@ private void lateInit() {
                 properties.setProperty(DOCKER_IMAGE_USER, namespace);
             }
             if (!properties.contains(PlatformMode.FABRIC8_EFFECTIVE_PLATFORM_MODE)) {
-                properties.setProperty(PlatformMode.FABRIC8_EFFECTIVE_PLATFORM_MODE, platformMode.toString());
+                properties.setProperty(PlatformMode.FABRIC8_EFFECTIVE_PLATFORM_MODE, runtimeMode.toString());
             }
         }
 
         openShiftConverters = new HashMap<>();
         openShiftConverters.put("ReplicaSet", new ReplicSetOpenShiftConverter());
         openShiftConverters.put("Deployment",
-            new DeploymentOpenShiftConverter(platformMode, getOpenshiftDeployTimeoutSeconds()));
+            new DeploymentOpenShiftConverter(runtimeMode, getOpenshiftDeployTimeoutSeconds()));
```

Comment:
Do we still need this converters ? I though the PR was all about getting rid of them